homeassistant/components/wirelesstag.py

```diff
@@ -271,7 +271,7 @@ def update(self):
     def device_state_attributes(self):
         """Return the state attributes."""
         return {
-            ATTR_BATTERY_LEVEL: self._tag.battery_remaining,
```

Comment:
Do not include the percent sign for `ATTR_BATTERY_LEVEL`.

Reply:
Updated.

Reply:
I think you then need to do the multiplication by hand?

Reply:
Updated...thank you for your suggestions, I'm very new to Python :)

Reply:
Okay, but please at least test before you push out updates :)

I suggest `int(self._tag.battery_remaining*100)` so we get completely rid of that decimal point.

Please confirm that this is consistent with your other devices (I only have one kind of battery device myself).

Reply:
Confirmed.  This is consistent, and this is consistent with WirelessTag's example client.

From: http://wirelesstag.net/jshtmlview.aspx?html=index.html&js=styles/client.js

```javascript
batDisplayPc.text((tag.batteryRemaining * 100).toFixed(0));
```